.github/dependabot.yml

```diff
@@ -1,6 +1,9 @@
 version: 2
 updates:
-  - package-ecosystem: "npm"
+  - package-ecosystem: "yarn"
```

Comment:
i tried `yarn` before, but apparently it's not valid. should be set back to `npm`.